src/pages/TimeTable.js

```diff
@@ -83,18 +83,20 @@ function TimeTable({ eventList, handleNewCustomEvent, handleDeleteEvent }) {
 
   // deletes custom event on click
   const handleDelete = (clickInfo) => {
-    console.log(clickInfo.event.start.toISOString());
-    if (
-      window.confirm(
-        `Are you sure you want to delete the event '${clickInfo.event.title}'`,
-      )
-    ) {
-      clickInfo.event.remove();
-      handleDeleteEvent(
+
+    const aimsTitle = aimsCourse.identifiedCourses.find(course => course === clickInfo.event.title)
```

Comment:
This is not going to work in the newer version of the extension, where the course title also includes the course name.